reactor-core/src/main/java/reactor/core/publisher/FluxProcessor.java

```diff
@@ -202,7 +202,7 @@ public Context currentContext() {
 	 * Create a {@link FluxProcessor} that safely gates multi-threaded producer
 	 * {@link Subscriber#onNext(Object)}.
 	 *
-	 * @reactor.discard The resulting processor discards elements received from the source
+	 * <p><strong>onDiscard Support:</strong> The resulting processor discards elements received from the source
```

Comment:
the feature is exposed to users as `doOnDiscard` rather than `onDiscard` so maybe this should be used in the `<strong>` header everywhere? This was a shortcoming of the original tag description that we can fix here.

Reply:
alternatively, simply use `Discard Support`?

Reply:
Rebased and updated to `Discard Support` which is more in line with the `Error More Support` "sentence".